apps/react-workshop/.ladle/global.css

```
@@ -27,3 +27,8 @@ main {
   position: relative;
   transform: translateZ(0);
 }
+
+/* TODO: Remove when the sidenav scroll works out-of-the-box in newer versions of Chrome. See #2283 */
```

I found a bug report in the ladle report which we can link to instead.

```suggestion
/* TODO: Remove when the sidenav scroll is fixed in Chrome. See https://github.com/tajo/ladle/issues/574 */
```